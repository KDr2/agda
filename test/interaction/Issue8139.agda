-- Andreas, 2025-10-25, issue #8139
-- Allow copattern-catchalls for the sake of splitting.

-- {-# OPTIONS -v interaction.case.filter:20 #-}
-- {-# OPTIONS -v tc.cover:30 #-}

open import Agda.Builtin.Sigma

postulate
  A : Set
  B : A → Set
  a : A
  b : B a

test : Σ A B
test .fst = a
test = {!!}  -- C-c C-c RET
