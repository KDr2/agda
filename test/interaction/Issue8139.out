(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals, Errors, Warnings*" "?0 : Σ A B ———— Error ————————————————————————————————————————————————— Issue8139.agda:16.1-17.5: error: [CoverageIssue] Incomplete pattern matching for test. Missing cases: test .snd when checking the definition of test ———— Warnings —————————————————————————————————————————————— Issue8139.agda:17.1-12: warning: -W[no]UnreachableClauses Unreachable clause when checking the definition of test" nil)
((last . 1) . (agda2-goals-action '(0)))
((last . 2) . (agda2-make-case-action '("test .snd = ?")))
((last . 1) . (agda2-goals-action '(0)))
